recipe/meta.yaml

```diff
@@ -25,35 +25,37 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
+    - debugpy
+    - comm
+    - hatchling
+    - ipython
+    - jupyter_client
+    - packaging
+    - psutil
   run:
```

Comment:
The `jupyter-core` requirement appears to be missing: https://github.com/ipython/ipykernel/blob/v6.25.0/pyproject.toml#L33

Reply:
It's included now